tutorials/migration.md

```diff
@@ -292,134 +237,112 @@ To get a list of all currently installed items run:
+For an apt-get installed openHAB 2 the `<openHAB 2 conf>` folder is located in `/etc/openhab2`. 
+For manually installed openHAB 1.x this file is located in `<openhab home>/configurations` and in manually installed openHAB 2 it is `<openhab home>/conf`.
 
-This file will have a lot of redundant configuration information in it that you
-have already moved over to individual binding .cfgs. Make sure to comment those
-out and just leave the config parameters for the bindings you are manually
-installing.
+This file will have a lot of redundant configuration information in it that you have already moved over to individual binding .cfgs. 
+Make sure to comment those out and just leave the config parameters for the bindings you are manually installing.
 
-Now copy the add-on's jar files that you want to install to the `<openHAB conf>/addons` folder for
-openHAB 2. For example:
+Now copy the add-on's jar files that you want to install to the `<openHAB conf>/addons` folder for openHAB 2. 
+For example:
 
 ```bash
 cp /usr/share/openhab/addons/org.openhab.binding.astro-1.8.3.jar /usr/share/openhab2/addons
 ```
 
-There may be some errors about there not being a `bind` and `unbind` method. If
-everything worked you should see in the log the same entries from the binding
-you see when openHAB 1.x starts.
+There may be some errors about there not being a `bind` and `unbind` method. 
+If everything worked you should see in the log the same entries from the binding you see when openHAB 1.x starts.
 
-Watch the logs for errors as they can be informative. For example, some Actions
-require a corresponding Binding be installed first. An example error of this
-sort looks like:
+Watch the logs for errors as they can be informative. 
+For example, some Actions require a corresponding Binding be installed first. 
+An example error of this sort looks like:
 
 ```text
 2016-09-08 15:15:04.613 [WARN ] [org.apache.felix.fileinstall        ] - Error while starting bundle: file:/openhab/addons/org.openhab.action.astro-1.8.3.jar
 org.osgi.framework.BundleException: Could not resolve module: org.openhab.action.astro [210]
   Unresolved requirement: Import-Package: org.openhab.binding.astro.internal.calc
 ```
 
-Note, the above error is for illustration purposes. You should not be
-installing Astro this way.
+Note, the above error is for illustration purposes. 
+You should not be installing Astro this way.
+These instructions are only for add-ons that are not listed in PaperUI or the Karaf Console.
 
 ### Final Add-ons Installation Steps
 
-openHAB 2 has a different
-[folder layout]({{base}}/installation/linux.html#file-locations).
-Of particular node are the configuration folders and userdata folders. See the
-previous link for the location of these folders for your installtion.
+openHAB 2 has a different [folder layout]({{base}}/installation/linux.html#file-locations).
+Of particular node are the <openHAB 2 conf> folders and <openHAB 2 userdata> folders. 
```

Comment:
According to #189 we should use a consistent `$FOLDER` notation for all articles.